requirements-windows.txt

```diff
@@ -1,2 +1,3 @@
-wheels/Shapely-1.6.4.post1-cp27-cp27m-win32.whl
+wheels/Shapely-1.6.4.post1-cp36-cp36m-win_amd64.whl
+numpy==1.16.3  # rq.filter: <1.17.0
```

Comment:
I think, that this version will get uninstalled and overwritten later on, as it's also defined in the requirements,txt. So it has to be reinstalled manually afterwards or adapted in the requirements.txt before building the application.